_maps/map_files/RandomRuins/SpaceRuins/telecomns_returns.dmm

```diff
@@ -0,0 +1,11797 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aa" = (
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2";
+	tag = ""
+	},
+/turf/simulated/floor/bluegrid{
+	name = "Mainframe Base";
+	nitrogen = 400;
+	oxygen = 0;
+	temperature = 80
+	},
+/area/ruin/space/telecomns/chamber)
+"an" = (
+/mob/living/simple_animal/hostile/malf_drone,
+/turf/space,
+/area/space)
+"ao" = (
+/obj/structure/sign/securearea{
+	desc = "A warning sign which reads 'LETHAL TURRETS'. Enter at your own risk!";
+	name = "LETHAL TURRETS";
+	pixel_x = 32;
+	pixel_y = 0
+	},
+/obj/machinery/porta_turret/ai_turret{
+	check_synth = 1;
+	lethal = 1;
+	faction = "malf_drone";
+	ailock = 1;
+	check_borgs = 1
+	},
+/obj/effect/turf_decal/box/white,
+/turf/simulated/floor/plasteel,
+/area/ruin/space/telecomns/foyer)
+"ax" = (
+/obj/structure/lattice,
+/obj/structure/grille/broken,
+/turf/space,
+/area/space/nearstation)
+"aN" = (
+/obj/machinery/light/small{
+	dir = 8
+	},
+/obj/structure/sign/vacuum{
+	pixel_x = 0;
+	pixel_y = 32
+	},
+/obj/structure/cable{
+	d1 = 2;
+	d2 = 4;
+	icon_state = "2-4";
+	tag = ""
+	},
+/mob/living/simple_animal/hostile/malf_drone,
+/obj/effect/spawner/random_spawners/cobweb_left_frequent,
+/turf/simulated/floor/catwalk,
+/area/ruin/space/telecomns/powercontrol)
+"bb" = (
+/obj/machinery/teleport/hub,
+/turf/simulated/floor/plating,
+/area/ruin/space/telecomns/tele)
+"bE" = (
+/obj/machinery/light,
+/turf/space,
+/area/ruin/space/telecomns)
+"bI" = (
+/obj/structure/window/plasmareinforced{
+	dir = 8
+	},
+/turf/simulated/floor/plasteel,
+/area/ruin/space/telecomns/tele)
+"bJ" = (
+/obj/effect/abstract/loot_trap,
+/turf/simulated/floor/bluegrid{
+	icon_state = "dark";
+	name = "Mainframe Floor";
+	nitrogen = 400;
+	oxygen = 0;
+	temperature = 80
+	},
+/area/ruin/space/telecomns/chamber)
+"bO" = (
+/obj/machinery/mech_bay_recharge_port{
+	dir = 1
+	},
+/turf/simulated/floor/bluegrid{
+	icon_state = "dark";
+	name = "Mainframe Floor";
+	nitrogen = 400;
+	oxygen = 0;
+	temperature = 80
+	},
+/area/ruin/space/telecomns/chamber)
+"bP" = (
+/obj/structure/table,
+/obj/effect/spawner/lootdrop/telecomns_core_table,
+/turf/simulated/floor/plasteel{
+	dir = 1;
+	icon_state = "vault";
+	name = "Mainframe floor";
+	nitrogen = 400;
+	oxygen = 0;
+	tag = "icon-vault (NORTH)";
```

Comment:
what are these for?

Reply:
Likely outdated variables from long ago, from the original floor of the original sat. Will remove